esp/platform/espcontext.cpp

```diff
@@ -454,6 +454,15 @@ class CEspContext : public CInterface, implements IEspContext
             m_txSummary->append("total", m_processingTime, "ms");
         }
     }
+    virtual void addTraceSummaryCumulativeTime(const char* name, unsigned __int64 time)
+    {
+        if (m_txSummary)
+            m_txSummary->updateTimer(name, time);
```

Comment:
This method was added for consistency - the context can add key value pairs, so it seemed appropriate to allow timer updates. Common usage has been to get and retain timer instances rather than using this method.